From: KevinDuan <duankaiwen@kylinos.cn>
Date: Mon, 17 Jun 2024 07:23:17 +0000
Subject: Translated using Weblate (Uyghur)

Currently translated at 94.4% (120 of 127 strings)

Translation: openkylin-nile-new/kom-service-support
Translate-URL: http://weblate.openkylin.top/projects/openkylin-nile-new/kom-service-support/ug/
---
 plugins/service-support/translations/kom-service-support_ug.ts | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/plugins/service-support/translations/kom-service-support_ug.ts b/plugins/service-support/translations/kom-service-support_ug.ts
index c913b61..68d0248 100644
--- a/plugins/service-support/translations/kom-service-support_ug.ts
+++ b/plugins/service-support/translations/kom-service-support_ug.ts
@@ -221,7 +221,7 @@
     <message>
         <location filename="../UI/uimainpage.cpp" line="85"/>
         <source>Multi-channel technical support services</source>
-        <translation type="unfinished"></translation>
+        <translation>Technischer Multi-Channel-Support</translation>
     </message>
     <message>
         <location filename="../UI/uimainpage.cpp" line="88"/>
@@ -309,17 +309,17 @@
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="102"/>
         <source>Has the issue been resolved?</source>
-        <translation type="unfinished"></translation>
+        <translation>Wurde das Problem behoben?</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="105"/>
         <source>Once identified, the issue will be closed and no further action will be taken.</source>
-        <translation type="unfinished"></translation>
+        <translation>Sobald das Problem identifiziert wurde, wird es geschlossen und es werden keine weiteren Maßnahmen ergriffen.</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="107"/>
         <source>resolved</source>
-        <translation type="unfinished"></translation>
+        <translation>entschlossen</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="109"/>
@@ -528,7 +528,7 @@
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="237"/>
         <source>Files</source>
-        <translation type="unfinished"></translation>
+        <translation>ھۆججەت</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="686"/>
